libcontainer/seccomp/seccomp_linux.go

```diff
@@ -135,6 +118,67 @@ func InitSeccomp(config *configs.Seccomp) (int, error) {
 	return seccompFd, nil
 }
 
+type unknownFlagError struct {
+	flag specs.LinuxSeccompFlag
+}
+
+func (e *unknownFlagError) Error() string {
+	return "seccomp flag " + string(e.flag) + " is not known to runc"
```

Comment:
not known? Not supported seems a more standard way to say this

Reply:
I think I copy-pasted that from the earlier code. And yes, I agree that "not supported" seems more common language, OTOH "more known" is more specific in this particular case.

"Not supported" is sort of generic, and can mean multiple things, such as the support for this feature or flags is
 - not yet added;
 - dropped;
 - not compiled it;
 - is impossible because of the current hardware or the kernel;
 and so on.
 
"Not known" is more specific, meaning this runc version is not aware that such flag exists (so, if it is exists, this is a clear hint that the runc version used is too old).

Reply:
So, if you take a look at the code adding flags to `runc features`, you'll see there are "known" and "supported" set of flags (with supported being a subset of known). This is why I'm using the word "known" here.

Reply:
:+1: